app/assets/javascripts/Components/Result/annotation_manager.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 
 export class AnnotationManager extends React.Component {
-  componentDidMount() {
+  componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
In general, omit right-most parameters if they aren't used. Here we can omit all three, and just do `componentDidUpdate()`.